client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -102,28 +102,32 @@ export function SitesDashboard( { queryParams: { search, status = 'all' } }: Sit
 			</PageHeader>
 			<PageBodyWrapper>
 				<>
-					<FilterBar>
-						<SitesSearch
-							searchIcon={ <SitesSearchIcon /> }
-							onSearch={ ( term ) => handleQueryParamChange( 'search', term?.trim() ) }
-							isReskinned
-							placeholder={ __( 'Search by name or domain…' ) }
-							disableAutocorrect={ true }
-							defaultValue={ search }
-						/>
-						<SelectDropdown selectedText={ selectedStatus.title }>
-							{ statuses.map( ( { name, title, count } ) => (
-								<SelectDropdown.Item
-									key={ name }
-									selected={ name === selectedStatus.name }
-									count={ count }
-									onClick={ () => handleQueryParamChange( 'status', 'all' !== name ? name : '' ) }
-								>
-									{ title }
-								</SelectDropdown.Item>
-							) ) }
-						</SelectDropdown>
-					</FilterBar>
+					{ allSites.length > 0 ? (
```

Comment:
We can short-circuit the expression with `&&` instead of a ternary so it's shorter. React does not render falsy values so it will just ignore the expression:

```suggestion
					{ allSites.length > 0 && (
```

And exclude the `: ''` bit at the bottom.

Reply:
> We can short-circuit the expression with `&&` instead of a ternary so it's shorter. React does not render falsy values so it will just ignore the expression:

@zaguiini Hot tip, thanks! Applied with 8a8bf05170